### DISCUSSION TITLE
Running satisfactory-server on truenas scale dragonfish

### DISCUSSION THREAD

**Discussion by Santana-2** [wolveix/satisfactory-server#303] [Q&A] [answered]
Hei Guys

During 0.8 I installed the truecharts version of this great server here. But now truecharts stopped providing support for truenas cale and I was not able to get the server running on 1.0. 

I decided to try to install the satisfactory-server with the "Custom App" Menu on my scale machine. Within this Menu I cann specify the image to pull (I used "wolveix/satisfactory-server", which works well). Additionally I specified the ports 7777 for TCP and UDP. I also set up a new dataset called "satisfactoryserver" on my machine and gave it root permissions. I then proceeded to bind this dataset within the Custom App Menu:  Host Path and Mount path are configured as: .../.../apps/satisfactoryserver

With this setup the server starts and I can access it within the Server Manager of the game. I can play on the server normal and also save the game. However, when I restat the app on scale, the server resets completely and I loose all saves. I cannot figure out what I did wrong, as I am not a very experienced user. Could anybody help me out?

**Comment by wolveix** [comment]
Update to the latest image, this was an issue when 1.0 first dropped: #266

Also, to be clear, have you bound the containers `/config` directory to your host?

**Reply by Santana-2** [reply]
Could you elaborate what you mean with containers /config? I did bind one host path volume  to this 
Host Path: .../.../apps/satisfactory-server
Mount Path: .../.../apps/satisfactory-server/config

Do I need to bind other directories as well? Also as host path or as a normal volume? (I have two different options in the GUI)

**Reply by wolveix** [reply, marked as answer]
Your mount path should be just `/config` :)

**Reply by Santana-2** [reply]
Thank you, I changed the Mount path to ` /config`

It works! Probably everyone reading this will shake their heads...Thank you so much!

**Reply by wolveix** [reply]
No worries! Enjoy, my friend!
